Plugin~/README.md

```diff
@@ -1,157 +1,158 @@
-# ネイティブプラグイン
+# Native Plugin
 
-`com.unity.webrtc` はネイティブプラグインを使用しています。
-ネイティブプラグインのビルド及び配置方法について説明します。
+This guide will cover building and deploying the native plugin `com.unity.webrtc` depends on.
 
-## 開発環境
+## Developing environment
 
-開発環境を構築するため、依存ライブラリをインストールします。
+Install dependencies to make development environment.
 
-### 依存ライブラリのインストール (Windows)
+### How to install dependencies (Windows)
 
-Windows 環境では、インストールに [chocolatey](https://chocolatey.org/) を使用します。
+On windows, [chocolatey](https://chocolatey.org/) is used to install.
 
 ```powershell
-# install CUDA
+# Install CUDA
 choco install cuda --version=10.1
 
-# install Windows SDK
+# Install Windows SDK
 choco install -y vcredist2010 vcredist2013 vcredist140 windows-sdk-10-version-1809-all
 
-# install Vulkan
+# Install Vulkan
 choco install -y wget
 wget https://vulkan.lunarg.com/sdk/download/1.1.121.2/windows/VulkanSDK-1.1.121.2-Installer.exe -O C:/Windows/Temp/VulkanSDK.exe
 C:/Windows/Temp/VulkanSDK.exe /S
 
+#install googletest
+cd %SOLUTION_DIR%
+git clone https://github.com/google/googletest.git
+cd googletest
+git checkout 2fe3bd994b3189899d93f1d5a881e725e046fdc2
+cmake . -G "Visual Studio 15 2017" -A x64 -B "build64"
+cmake --build build64 --config Release
+mkdir include\gtest
+xcopy /e googletest\include\gtest include\gtest
+mkdir include\gmock
+xcopy /e googlemock\include\gmock include\gmock
+mkdir lib
+xcopy /e build64\googlemock\Release lib
+xcopy /e build64\googlemock\gtest\Release lib
+
+# Install CMake 3.18.0
+choco install cmake -y --version 3.18.0
```

Comment:
I think this line above `install googletest` section.
camek use in install googletest section

Reply:
fixed it